SPECS/intel-igc.spec

```
@@ -49,6 +49,9 @@ version %{kernel_version}.
 %autosetup -p1 -n %{name}-%{version}
 %{?_cov_prepare}
 
+# Set module version to upstream kernel release
+/usr/bin/echo '#define DRV_VERSION "%{version}"' >> igc.h
```

And sed substitution here:
```suggestion
sed -e 's|@@DRV_VERSION@@|%{version}|' -i igc.h
```